src/main/java/step2/domain/Lotto.java

```diff
@@ -0,0 +1,64 @@
+package step2.domain;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Lotto {
+    private final int LOTTO_SIZE = 6;
+    private final int MINIMUN = 1;
+    private final int MAXIMUN = 45;
+
+    private List<Integer> numbers;
+
+    private Lotto(List<Integer> numbers) {
+        validation(numbers);
+        this.numbers = numbers;
+    }
+
+    public static Lotto from(List<Integer> list) {
+        return new Lotto(list);
+    }
+
+    private void validation(List<Integer> list) {
+        if (list.size() != LOTTO_SIZE) {
+            throw new IllegalArgumentException("로또 길이는 6개 이어야 합니다.");
```

Comment:
```suggestion
            throw new IllegalArgumentException("로또 길이는 " + LOTTO_SIZE + "개 이어야 합니다.");
```

상수가 변경 됐을 때 메시지를 신경쓰지 않도록 메시지에서 상수를 활용할 수 있습니다 :)